plugins/modules/zos_operator_action_query.py

```diff
@@ -55,6 +56,17 @@
       - A trailing asterisk, (*) wildcard is supported.
     type: str
     required: false
+  msg_filter:
+    description:
+        - Return outstanding messages requiring operator action awaiting a
+          reply that match a regex filter.
+        - If the message filter is not specified, all outstanding messages
+          are returned regardless of their content.
+        - Valid Python regular expressions are supported. See L(the official 
+          documentation,https://docs.python.org/3/library/re.html) for more information.
```

Comment:
Just a comment no need for change, I pointed my browser at "https://docs.python.org/library/re.html" and it resolved to "https://docs.python.org/3/library/re.html" somehow the latest which we might want to consider in the future if we are editing this module.